Source/lighting.cpp

```diff
@@ -350,13 +350,15 @@ void MakeLightTable()
 		// Blood wall lighting
 		const auto shades = static_cast<int>(LightTables.size() - 1);
 		for (int i = 0; i < shades; i++) {
-			auto &lightTable = LightTables[i];
-			constexpr int range = 16;
-			for (int j = 0; j < range; j++) {
-				uint8_t color = ((range - 1) << 4) / shades * (shades - i) / range * (j + 1);
+			uint8_t &lightTable = LightTables[i];
+			constexpr int Range = 16;
+			for (int j = 0; j < Range; j++) {
+				uint8_t color = ((Range - 1) << 4) / shades * (shades - i) / Range * (j + 1);
 				color = 1 + (color >> 4);
-				lightTable[j + 1] = color;
-				lightTable[31 - j] = color;
+				int idx = j + 1;
+				lightTable[idx] = color;
+				idx = 31 - j;
+				lightTable[idx] = color;

```

Comment:
You aren't modifying anything here, can just do it inline.
```suggestion
				lightTable[j + 1] = color;
				lightTable[31 - j] = color;
```